package/MDAnalysis/analysis/contacts.py

```diff
@@ -1150,8 +1149,6 @@ def load(self, filename):
     def plot(self, filename=None, **kwargs):
         """Plot q(t).
 
-        .. function:: ContactAnalysis1.plot([filename, ...])
-
         If *filename* is supplied then the figure is also written to file (the
```

Comment:
filename could be in backtics because it refers to a function parameter... just nit-ticking, er, nitpicking.